### PR TITLE
card scroll

### PR DESCRIPTION
**Pull request by aragideti**

### CODE REVIEW

File: app/src/main/java/com/compose/flyticketsbooking/feature/home/HomeScreen.kt [hunk 1]
```diff
@@ -27,22 +37,84 @@ fun HomeScreen(
...
-        modifier = Modifier.fillMaxSize(),
-        contentAlignment = Alignment.BottomStart
+    ConstraintLayout(
+        modifier = Modifier
+            .fillMaxSize()
+            .verticalScroll(rememberScrollState())
+
     ) {
-        Text(text = "Hot offer", style = MaterialTheme.typography.h1)
+        val (hotOfferText, cardSlider, seeAllText, bookFLightText, menuButton, tabSwitcher)
```
Comment (slothdeboss) [inline app/src/main/java/com/compose/flyticketsbooking/feature/home/HomeScreen.kt:46]: @aragideti I think you need to wrap `createRefs()` into `remember { }` block

File: app/src/main/java/com/compose/flyticketsbooking/feature/home/HomeScreen.kt [hunk 2]
```diff
@@ -27,22 +37,84 @@ fun HomeScreen(
...
+                start.linkTo(parent.start)
+                end.linkTo(parent.end)
+                top.linkTo(parent.top, margin = 4.dp)
+                height = Dimension.wrapContent
+            }
+        )
+    val (selected, setSelected) = remember {
+        mutableStateOf(0)
+    }
+        val items = listOf("One way", "Round", "Multiply")
```
Comment (slothdeboss) [inline app/src/main/java/com/compose/flyticketsbooking/feature/home/HomeScreen.kt:72]: @aragideti maybe it's better to extract these values into resources or enum?

File: app/src/main/java/com/compose/flyticketsbooking/feature/home/HomeScreen.kt [hunk 3]
```diff
@@ -27,22 +37,84 @@ fun HomeScreen(
...
+            modifier = Modifier.constrainAs(bookFLightText) {
+                start.linkTo(parent.start)
+                end.linkTo(parent.end)
+                top.linkTo(parent.top, margin = 4.dp)
+                height = Dimension.wrapContent
+            }
+        )
+    val (selected, setSelected) = remember {
+        mutableStateOf(0)
+    }
```
Comment (slothdeboss) [inline app/src/main/java/com/compose/flyticketsbooking/feature/home/HomeScreen.kt:71]: @aragideti also if you want to mutate these values I think you need to change from `val` to `var`

File: app/src/main/java/com/compose/flyticketsbooking/feature/home/tabs/TabIndicator.kt [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+        text = text,
+        color = tabTextColor,
+        textAlign = TextAlign.Center,
+        fontSize = 16.sp,
+    )
+}
+
+@Composable
+fun CustomTabSample(
+    modifier: Modifier = Modifier,
```
Comment (slothdeboss) [inline app/src/main/java/com/compose/flyticketsbooking/feature/home/tabs/TabIndicator.kt:65]: @aragideti modifier should be the first parameter with a default value, but if you have any other parameters with no default value, they should be before modifier. In this case the order should be `setSelected, selected, items, modifier`

File: app/src/main/java/com/compose/flyticketsbooking/feature/home/tabs/TabIndicator.kt [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+    )
+}
+
+@Composable
+private fun TabRounded(
+    selectedItemIndex: Int,
+    items: List<String>,
+    modifier: Modifier = Modifier,
+    tabWidth: Dp = 120.dp,
+    onClick: (index: Int) -> Unit,
```
Comment (slothdeboss) [inline app/src/main/java/com/compose/flyticketsbooking/feature/home/tabs/TabIndicator.kt:110]: @aragideti same here, please fix the order

**Review by slothdeboss — CHANGES_REQUESTED** [review]

**Review by igel911 — APPROVED** [review]

**Review by slothdeboss — APPROVED** [review]

**Pull request merged**